### PR TITLE
Updates APIs in README

### PR DESCRIPTION
**Pull request by HeshamMegid**
## Summary of Changes

* Updates the readme to reflect changes to our plan for 1.0
* Separates APIs into sections according to which class they are in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,35 +20,78 @@ A Flutter plugin for [Instabug](https://instabug.com/).
...
+|                                                                                             | `setReportTypes(@BugReporting.ReportType int... types)`<br>`promptOptionsEnabledReportTypes`                                |
+|                                                                                             | `setExtendedBugReportState(ExtendedBugReport.State state)`<br>`extendedBugReportMode`                                       |
+|                                                                                             | `setOptions(@Option int... options)`<br>`bugReportingOptions`
+|                                                                                             | `show(@BugReporting.ReportType int type)`<br>`+ showWithReportType:options:`
+
+#### `InstabugLog`
+
+| API Method                                    | Native Equivalent (Android/iOS)                              |
+|-----------------------------------------------|--------------------------------------------------------------|
+| `Instabug.logDebug(String message)            | `InstabugLog.d(String message)`<br>`+ [IBGLog logDebug:]`    |
```
Comment (alyezz) [inline README.md]: Missing a ` at the end there

File: README.md [hunk 2]
```diff
@@ -20,35 +20,78 @@ A Flutter plugin for [Instabug](https://instabug.com/).
...
+| `Instabug.logVerbose(String message)`         | `InstabugLog.v(String message)`<br>`+ [IBGLog logVerbose:]`  |
+| `Instabug.logInfo(String message)`            | `InstabugLog.i(String message)`<br>`+ [IBGLog logInfo:]`     |
+| `Instabug.logWarn(String message)`            | `InstabugLog.w(String message)`<br>`+ [IBGLog logWarn:]`     |
+| `Instabug.logError(String message)`           | `InstabugLog.e(String message)`<br>`+ [IBGLog logError:]`    |
+
+#### `Surveys`
+
+| API Method                                    | Native Equivalent (Android/iOS)                                                                                                             |
+|-----------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+|                                               | `setState(Feature.State state)`<br>``enabled`                                                                                               |
```
Comment (alyezz) [inline README.md]: An extra `
`enabled`

File: README.md [hunk 3]
```diff
@@ -20,35 +20,78 @@ A Flutter plugin for [Instabug](https://instabug.com/).
...
+| `getUserAttributeForKey(Sring Key)`                                  | `getUserAttribute(String key)`<br>`+ userAttributeForKey:`                                                |
+| `removeUserAttributeForKey(String key)`                              | `removeUserAttribute(String key)`<br>`+ removeUserAttributeForKey:`                                       |
+| `getUserAttributes()`                                                | `getAllUserAttributes()`<br>`+ userAttributes:`                                                           |
+| `logUserEventWithName(String name)`                                  | `logUserEvent(String name)`<br>`+ logUserEventWithName:`                                                  |
+| `show()`                                                             | `show()`<br>`+ show`                                                                                      |
+| `clearAllLogs(String message)`                                       | `clearLogs()`<br>`+ clearAllLogs:`                                                                        |
```
Comment (alyezz) [inline README.md]: I think clearAllLogs should be under InstabugLog

File: README.md [hunk 4]
```diff
@@ -72,17 +71,18 @@ The section below contains the APIs we're planning to implement for our 1.0 rele
 
 | API Method                                    | Native Equivalent (Android/iOS)                              |
 |-----------------------------------------------|--------------------------------------------------------------|
-| `Instabug.logDebug(String message)            | `InstabugLog.d(String message)`<br>`+ [IBGLog logDebug:]`    |
-| `Instabug.logVerbose(String message)`         | `InstabugLog.v(String message)`<br>`+ [IBGLog logVerbose:]`  |
-| `Instabug.logInfo(String message)`            | `InstabugLog.i(String message)`<br>`+ [IBGLog logInfo:]`     |
-| `Instabug.logWarn(String message)`            | `InstabugLog.w(String message)`<br>`+ [IBGLog logWarn:]`     |
-| `Instabug.logError(String message)`           | `InstabugLog.e(String message)`<br>`+ [IBGLog logError:]`    |
+| `logDebug(String message)                     | `d(String message)`<br>`+ logDebug:`                         |
```
Comment (alyezz) [inline README.md]: Still missing an ` 

`logDebug(String message)`

Comment (HeshamMegid) [reply README.md]: 🤦‍♂️ we need a markdown linter.

**Pull request merged**